docker-compose.yml

```diff
@@ -160,6 +161,12 @@ services:
       POSTGRES_USER: postgres
       POSTGRES_PASSWORD: postgres
 
+  postgres-persisted:
+    extends:
+      service: postgres
+    volumes:
+      - "./pg/db:/var/lib/postgresql/data/"
+
```

Comment:
Maybe we should also remove the ports for `postgres-dev` to avoid conflicts with local postgres instances?

Reply:
My understanding is that the `-dev` stuff is for developers who did not install all the native stuff, so there shouldn’t be a problem in this case. Exposing the port gives them the possibility to connect with their favourite postgres client. I don’t have a strong opinion though, as I don’t think anyone is currently using this kind of setup.

Reply:
True, let's leave it like this then.